lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,27 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let kTypeStrippingMode;
+ObjectDefineProperty(process.features, 'typescript', {
+  __proto__: null,
+  get() {
+    if (kTypeStrippingMode === undefined) {
+      if (getOptionValue('--experimental-transform-types')) {
+        kTypeStrippingMode = 'transform';
+      } else if (getOptionValue('--experimental-strip-types')) {
+        kTypeStrippingMode = 'strip';
+      } else {
+        kTypeStrippingMode = '';
```

Comment:
```suggestion
        kTypeStrippingMode = '';
```
Empty string would probably be odd, maybe leave it undefined?